src/utils/global_stats_alloc/src/lib.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright 2022 Singularity Data
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use stats_alloc::StatsAlloc;
+use tikv_jemallocator::Jemalloc;
+
+pub static INSTRUMENTED_JEMALLOC: StatsAlloc<Jemalloc> = StatsAlloc::new(Jemalloc);
```

Comment:
I think this is probably a very bad idea and potentially the source of contention. We may want to find a cheaper thread-local way collecting allocation stats than rely on a global atomic var.

Reply:
Anw, we should benchmark this on a read-heavy workload with a lot of operator cache misses.

Reply:
Agree... Can be a potential performance killer.